scripts/lib/setup/installers.sh

```diff
@@ -374,6 +427,7 @@ function use_android_sdk() {
     local ANDROID_PLATFORM_VERSION=$(toolversion ANDROID_PLATFORM_VERSION)
     touch ~/.android/repositories.cfg
     echo y | sdkmanager "platform-tools" "build-tools;$ANDROID_BUILD_TOOLS_VERSION" "platforms;$ANDROID_PLATFORM_VERSION"
+    sdkmanager --licenses
```

Comment:
Missing `yes` for `stkmanager --licenses`.

Reply:
it's there in the latest push